tf_big/python/tensor.py

```diff
@@ -167,6 +177,10 @@ def _convert_tensorflow_tensor(tensor):
 
 def convert_to_tensor(tensor):
   if isinstance(tensor, Tensor):
+    # Big tensor ops expect tensors of rank >= 2
+    rank = tensor._raw.shape.rank
+    if rank in (0, 1):
+      tensor._raw = tf.reshape(tensor._raw, [1,1])
```

Comment:
I had to reshape to handle the following scenario in Paillier:
```
n = tf_big.convert_to_tensor(n)
self.g = 1 + n
```
Here n is scalar of shape (). Because of 1 gets lifted to a tensor of shape (1,1), tf_big.add throw an error because the shapes don't match.